src/main/java/racingcar/view/InputView.java

```diff
@@ -6,21 +6,17 @@
 
     private static final String INPUT_GUIDE_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
     private static final String ASK_TRY_COUNT = "시도할 횟수는 몇 회인가요?";
-    private static final Scanner scanner = new Scanner(System.in);
+    private static final Scanner SCANNER = new Scanner(System.in);
 
-    public String[] inputCarName() {
+    public String inputCarNames() {
         System.out.println(INPUT_GUIDE_MESSAGE);
 
-        String userInput = scanner.nextLine();
-
-        return userInput.split(",");
+        return SCANNER.nextLine();
```

Comment:
이 부분은 제가 저번에 잘못 피드백을 드렸던 것 같아요. 🙇‍♂️ 
이전과 동일하게 두는 것이 더 좋을 것 같습니다. 👀 
(이래서 RacingGame 객체에 String이 입력되고 있는 거였군요...? 제 불찰입니다. 😭) 

```suggestion
        String userInput = scanner.nextLine();
        return userInput.split(",");
```